lib/api/dsl/index.js

```diff
@@ -129,11 +150,9 @@ function flattenObject(target, id) {
 
 function flattenStep(output, object, prev, delimiter) {
   Object.keys(object).forEach(key => {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Refactor this function to use "return" consistently. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS3801)